appveyor.yml

```diff
@@ -3,21 +3,19 @@ version: '{build}'
 skip_tags: true
 
 environment:
+  JRUBY_OPTS: "--dev -J-Xmx1024M --debug"
   matrix:
-    - ruby_version: "21"
-    - ruby_version: "21-x64"
-    - ruby_version: "jruby-9.0.4.0"
+    - ruby_version: "Ruby21"
+    - ruby_version: "Ruby21-x64"
+    - ruby_version: "jruby-9.0.0.0"
 
 cache:
   - vendor/bundle
 
 install:
-  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
-  - ruby --version
-  - gem --version
+  - SET PATH=C:\%ruby_version%\bin;%PATH%
   - gem install bundler
-  - bundler --version
-  - bundle platform
+  - bundle env
```

Comment:
Yeah, it will output more useful and correct information about its environments:

```
Environment
 
    Bundler   1.11.2
    Rubygems  2.4.8
    Ruby      2.2.2p0 (2015-07-21 revision 50293) [java]
    Git       2.7.1.windows.2
...
```

https://ci.appveyor.com/project/bf4/active-model-serializers/build/10/job/0846cedglubbskwo#L10

---------------------

`ruby --version` is not correct for JRuby because binary name is jruby so `jruby --version` is correct:

https://ci.appveyor.com/project/mtsmfm/active-model-serializers/build/3/job/k31toapvkg9osoli#L7

We don't have to take care about that to use `bundle env`.